csharp/autobuilder/Semmle.Autobuild.CSharp/StandaloneBuildRule.cs

```diff
@@ -17,36 +16,24 @@ internal StandaloneBuildRule(string? dotNetPath)
 
         public BuildScript Analyse(IAutobuilder<CSharpAutobuildOptions> builder, bool auto)
         {
-            BuildScript GetCommand()
+            if (!builder.Options.Buildless
```

Comment:
This is not completely logically equivalent compared to previously.
The part about `builder.Options.Buildless` seems odd to me as we have already guarded the creation of the `StandAloneBuildRule` by this.
Maybe just remove that part of the condition.